packages/shared-http-pieces/src/tsconfig.json

```diff
@@ -6,7 +6,9 @@
 		"outDir": "../dist",
 		"noEmit": false,
 		"incremental": true,
-		"tsBuildInfoFile": "../dist/.tsbuildinfo"
+		"tsBuildInfoFile": "../dist/.tsbuildinfo",
```

Comment:
Gotta add these flags to `packages/shared-http-pieces/tsconfig.eslint.json`  as well to make typechecking pass